sessions/js-forms/demo-start/css/styles.css

```
@@ -0,0 +1,62 @@
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+}
+
+body {
+	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif,
+		'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+	background-color: lightgray;
+}
+
+main {
+	display: flex;
+	flex-direction: column;
+	gap: 10px;
+	margin: auto;
+	padding: 20px;
+	width: 90%;
+	height: 100vh;
+	background-color: white;
+}
+
+h1 {
+	padding: 5px;
+	background-color: black;
+	color: white;
+}
+
+form {
+	display: flex;
+	flex-direction: column;
+	gap: 10px;
+}
+
+fieldset {
+	display: flex;
+	flex-direction: column;
+	gap: 10px;
+	padding: 10px;
+}
+
+legend {
+	background-color: black;
+	color: white;
+}
+
+button {
+	margin: auto;
+	width: fit-content;
+}
+
+input[type='submit'] {
+	margin: auto;
+	width: fit-content;
+}
+
+textarea {
+	width: fit-content;
+}
```

I would suggest the following css instead. The layout is a bit easier with grid and the textarea needs `width: 100%;` so that it doesn't bleed out of the form.
```suggestion
*,
*::before,
*::after {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}

body {
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
    Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
  background-color: lightgray;
}

main {
  display: grid;
  gap: 10px;
  margin: auto;
  padding: 20px;
  width: 90%;
  background-color: white;
}

h1 {
  padding: 5px;
  background-color: black;
  color: white;
}

form {
  display: grid;
  gap: 10px;
}

fieldset {
  display: grid;
  gap: 10px;
  padding: 10px;
}

legend {
  background-color: black;
  color: white;
}

button {
  margin: auto;
  width: fit-content;
}

input[type='submit'] {
  margin: auto;
  width: fit-content;
}

textarea {
  width: 100%;
}
```